docs/logql.md

```diff
@@ -0,0 +1,159 @@
+# LogQL: Log Query Language
+
+Loki comes with its very own language for querying logs called *LogQL*. LogQL
+can be considered a distributed `grep` with labels for extra querying control.
+
+A basic LogQL query consists of two parts: the **log stream selector** and a
+**filter expression**. Because of how Loki is designed, a set of log streams
+must be selected using a [Prometheus](https://prometheus.io)-style log stream
```

Comment:
Linking Prometheus in this specific context may be misleading, because the reader may expect to find details about the query syntax at the link, while we just link the homepage and we may have the reader getting lost, while (hopefully) all the answers to LogQL are below in this doc.